src/createStreamingManager.ts

```diff
@@ -121,17 +119,22 @@ export async function createStreamingManager<T extends CreateStreamOptions>(
                     rtcStats: data ?? [] 
                 })
             } else if (event === StreamEvents.StreamDone) {
+                analytics?.track('agent-video', {
+                    event,
+                    rtcStats: data ?? [] 
+                })
                 console.log('StreamDone');
             } else if (event === StreamEvents.StreamFailed) {
                 callbacks.onVideoStateChange?.(StreamingState.Stop, { event, data });
 
                 clearInterval(videoStatsInterval);
                 console.log('StreamFailed');
             } else {
-                // Better remove it, because it's too noisy, since message have too many partial messages
-                analytics?.track('agent-on-message-streaming', {
-                    event
-                });
+                if(event === StreamEvents.ChatAnswer) {
+                    analytics?.track('agent-on-message-streaming', {
```

Comment:
agent-message-recieved (the old event name we had_

Reply:
renamed